pkg/dss/auth/auth.go

```diff
@@ -229,9 +247,32 @@ func (a *Authorizer) AuthInterceptor(ctx context.Context, req interface{}, info
 		return nil, dsserr.PermissionDenied(fmt.Sprintf("missing scopes: %v", err))
 	}
 
+	if claims.Issuer == "" {
+		return nil, dsserr.Unauthenticated("missing issuer URI")
+	}
+
+	if err := a.checkExpired(claims.ExpiresAt); err != nil {
+		return nil, dsserr.Unauthenticated(fmt.Sprintf("%v", err))
+	}
+
 	return handler(ContextWithOwner(ctx, models.Owner(claims.Subject)), req)
 }
 
+func (a *Authorizer) checkExpired(claimedExpireTime int64) error {
```

Comment:
Can we add the checkExpired to the claims.Valid() ?
https://github.com/interuss/dss/blob/60d63fb8dc0eee095d688d4d9427961dfbce15f1/pkg/dss/auth/claims.go#L38

This is how it is done in jwt.Valid, so it makes sense to put it there.

Also a nit: we can use their expiredAt if you choose too https://github.com/dgrijalva/jwt-go/blob/master/claims.go#L69